core/templates/dev/head/profile/profile.html

```diff
@@ -41,6 +41,13 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
               </span>
             </p>
             <p>
+              <span ng-if="subjectInterests">
+                Subject Interests: <[subjectInterests]></span>
+              <span ng-if="!subjectInterests">
```

Comment:
I am inclined to have something on the profile page even if no subject interests are specified. It incentivizes users to at least add one, and it's an easy thing to do. 

How do you feel about something like "Subject Interests: not specified" if nothing is specified?